actioncable/test/javascript/src/unit/action_cable_test.coffee

```diff
@@ -22,3 +22,19 @@ module "ActionCable", ->
       document.head.removeChild(element)
 
       assert.equal consumer.url, testURL
+
+  module 'WebSocket', ->
+    test 'default is window.WebSocket', (assert) ->
+      assert.equal ActionCable.WebSocket, window.WebSocket
+
+    test 'configurable', (assert) ->
+      ActionCable.WebSocket = ''
+      assert.equal ActionCable.WebSocket, ''
+
+  module 'logger', ->
+    test 'default is window.console', (assert) ->
+      assert.equal ActionCable.logger, window.console
+
+    test 'configurable', (assert) ->
+      ActionCable.logger = ''
+      assert.equal ActionCable.logger, ''
```

Comment:
How about grouping all of these new tests in one `module "Adapters"`? And, I know this is nit picky, but let's use double quotes everywhere to match the existing style.